### PR TITLE
RE2022-296: get KB genome meta info

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: src/loaders/workspace_downloader/workspace_downloader.py [hunk 1]
```diff
@@ -117,12 +139,12 @@ def _process_object_info(obj_info, genome_upa):
...
-    res_dict[loader_common_names.FLD_KB_OBJ_UPA] = "{6}/{0}/{4}".format(*obj_info)
-    res_dict[loader_common_names.FLD_KB_OBJ_NAME] = obj_info[1]
-    res_dict[loader_common_names.FLD_KB_OBJ_TYPE] = obj_info[2]
-    res_dict[loader_common_names.FLD_KB_OBJ_TIMESTAMP] = obj_info[3]
-    res_dict[loader_common_names.FLD_KB_OBJ_GENOME_UPA] = genome_upa
+    res_dict = {loader_common_names.FLD_KB_OBJ_UPA: "{6}/{0}/{4}".format(*obj_info),
+                loader_common_names.FLD_KB_OBJ_NAME: obj_info[1], loader_common_names.FLD_KB_OBJ_TYPE: obj_info[2],
+                loader_common_names.FLD_KB_OBJ_TIMESTAMP: obj_info[3],
+                loader_common_names.FLD_KB_OBJ_GENOME_UPA: "{6}/{0}/{4}".format(*genome_info)}
+    res_dict.update(_get_genome_obj_meta(genome_info))
```
[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/loaders/workspace_downloader/workspace_downloader.py:149]: 👍

[the remark this answers is not in the document]
Comment (MrCreosote) [reply src/loaders/workspace_downloader/workspace_downloader.py:149]: Isn't there already a meta file? I was thinking we could just stick both meta in there?

[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/loaders/workspace_downloader/workspace_downloader.py]: 👍  In that case, I also moved the parsing/renaming logic to the parser (for later PRs).

**Review by MrCreosote — APPROVED** [review]
LGTM

**Pull request merged**